src/document-click.js

```diff
@@ -108,6 +113,20 @@ export function onDocumentElementClick_(e, viewport) {
   const win = doc.defaultView;
 
   const tgtLoc = parseUrl(target.href);
+
+  // On Safari iOS, custom protocol links will fail to open apps when the
+  // document is iframed - in order to go around this, we set the top.location
+  // to the custom protocol href.
+  const isSafariIOS = platform.isIos() && platform.isSafari();
+  const isEmbedded = win.parent && win.parent != win;
+  const isNormalProtocol = /^https?:$/.test(tgtLoc.protocol);
+  if (isSafariIOS && isEmbedded && !isNormalProtocol) {
+    win.open(target.href, '_blank');
```

Comment:
As a bug fix - we should just roll with `_blank`. If experience looks not great, let's file a bug to consider using `_top` instead. That would open up another layer or two of testing - hence the new bug. In the meantime, another question - what target do these URLs arrive now with from `cdn.ampproject.org`?